src/main/java/Brave.java

```diff
@@ -0,0 +1,64 @@
+import java.util.Scanner;
+import java.lang.String;
+
+public class Brave {
+    public static void main(String[] args) {
+        String input;
+        String[] splitInput;
```

Comment:
Plural form should be used on names representing a collection of objects - maybe use splitInputs instead?